institution.py

```diff
@@ -105,16 +105,22 @@ def get(self):
       preferences_enabled = serving_session.login_type == "preferences"
       verification_enabled = serving_session.login_type == "verification"
       schedule_enabled = serving_session.login_type == "schedule"
+      preregistration_enabled = serving_session.login_type == "preregistration"
+      postregistration_enabled = serving_session.login_type == "postregistration"
       if not serving_session.session_name == session.name:
         preferences_enabled = False
```

Comment:
I wonder if we would be better off just sending serving_session.session_name and serving_session.login_type to the client instead of computing all the booleans here on the back end.